README.md

```diff
@@ -160,7 +188,7 @@ buildTypes {
 }
 ```
 
-#### **Step 3** - Update Build Names
```

Comment:
I removed the asterisks here since they're redundant - Markdown formats all headers as bold by default